lib/folio_sync/archives_space_to_folio/folio_synchronizer.rb

```diff
@@ -10,33 +10,20 @@ class FolioSynchronizer
       def initialize(instance_key)
         @logger = Logger.new($stdout)
         @instance_key = instance_key
-
-        @saving_errors = []
-        @fetching_errors = []
-        @downloading_errors = []
-        @syncing_errors = []
-      end
-
-      def fetch_and_sync_resources_to_folio(last_x_hours)
-        @downloading_errors = []
-        @syncing_errors = []
-        modified_since = Time.now.utc - (ONE_HOUR_IN_SECONDS * last_x_hours) if last_x_hours
-
-        download_archivesspace_marc_xml(modified_since)
-        sync_resources_to_folio
+        clear_error_arrays!
       end
 
-      # WIP - new sync method
       def fetch_and_sync_aspace_to_folio_records(last_x_hours)
-        @fetching_errors = []
-        @saving_errors = []
-        @downloading_errors = []
-        @syncing_errors = []
+        clear_error_arrays!
```

Comment:
Are the error arrays cleared here each time because `fetch_and_sync_aspace_to_folio_records` might be called more than once on the same FolioSynchronizer instance?

Reply:
Good catch, I removed this line since `fetch_and_sync_aspace_to_folio_records` shouldn't be run more than once